pkg/cedar/intake/models/easi_biz_case.go

```diff
@@ -16,6 +16,7 @@ type EASIBizCase struct {
 	AsIsPros             *string                 `json:"asIsPros,omitempty" jsonschema:"description=Pros of the current solution,example=Well known workflows and end products"`
 	AsIsSummary          *string                 `json:"asIsSummary,omitempty" jsonschema:"description=Summary of the current solution,example=Managed through spreadsheets and email"`
 	AsIsTitle            *string                 `json:"asIsTitle,omitempty" jsonschema:"description=Name of the current solution,example=Spreadsheets and Email"`
+	BusinessCaseID       string                  `json:"businessCaseId" jsonschema:"description=Unique ID of this business case,example=91e5c1f3-11fb-4124-805c-adbdd02c5395"`
```

Comment:
I made this ID a required field which i'm pretty sure is the right call but it is making me second guess my thought process in making the business case ID field in the EASILifecycleCost struct optional.

We may be able to just remove that ID field altogether, thoughts?

Reply:
Actually, on second thought it might not make sense to have this field be required - did we agree to send the "skeleton" of the business case when the intake was first submitted? If so then I don't think there would be a business case ID at that point.

The point about deleting the ID in the EASILifecycleCost struct is still valid I think.